flamingpy/codes/graphs/egraph.py

```diff
@@ -203,4 +202,15 @@ def draw(self, **kwargs):
         """
         from flamingpy.utils.viz import draw_EGraph
 
-        return draw_EGraph(self, **kwargs)
+        _, ax = draw_EGraph(self, **kwargs)
+        return ax
+
+    def draw_adjacency(self, **kwargs):
+        """Draw the adjacency matrix with matplotlib.
+
+        See flamingpy.utils.viz.plot_mat_heat_map for more details.
+        """
+        from flamingpy.utils.viz import plot_mat_heat_map
+
+        adj = self.adj_mat
```

Comment:
```suggestion
        adj = self.adj_generator(sparse=False)
```
(Just in case self.adj_mat is None)